### ISSUE TITLE
backpack_0, backpack_1, backpack_2, backpack_3 all returning a value of 0

### ISSUE THREAD

**Issue by ritteraf** [odota/web#2729]
**Current Behavior**
When making a call to: GET /matches/{match_id}
Under players JSON
values of backpack_0, backpack_1, backpack_2, backpack_3 are all returning a value of 0 or null

**Expected behavior/code**
Should be returning the id of the associate item

**Environment**
XCode

**Possible Solution**
Most likely the addition of new items to the game, or the addition of the shard item has bugged the API

**Additional context/Screenshots**
Pretty self explanitory.

**Comment by ritteraf** [comment]
While you're at it, it seems that the docs do not show backpack_3. Most likely the docs were written before the addition of the new item slot in dota2.

**Comment by howardchung** [comment]
Aren't there only 3 backpack slots? What is the ID of the match you're looking at?

**Comment by ritteraf** [comment]
A few patches ago they added a new item slot for "neutral items". It's not technically a backpack slot, but up until recently the backpack_3 value was showing the hero's neutral item. Now I'm not getting a value for any backpack slot. It's the same on every match I query.

**Comment by howardchung** [comment]
Neutral items are in item_neutral. I think it's always been that way.

**Comment by howardchung** [comment]
and for this match anyway, backpack slots seem to be showing data:
https://api.opendota.com/api/matches/5790546095

**Comment by ritteraf** [comment]
Hi Howard, after you mentioning it - I am receiving data for item_neutral. I guessed since backpack_3 existed, I always assumed that's where you were storing the neutral item slot.

I am receiving a null value in the json response for backpack_3 in all queries. I tested 10 matches, and some players were showing values in their backpacks, so I'll assume that portion is functioning normally. I guess just seeing that null in backpack_3 threw me off. Thanks!
